docsk8s/administering/ls-k8s-stack-monitoring-cloud.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[ls-k8s-monitor-elastic-cloud]]
+==== Ship metrics to Elastic Cloud
+
+NOTE: The prerequisite of the example is having ECK installed. Checkout <<qs-set-up>>
```

Comment:
Is it a prerequisite to have ECK installed? Or just the CRDs?

Reply:
Do you mean from the perspective of whether the Beats CRD can be used in isolation, without any of the other components of the stack?

It feels like it might be a common use case to not want to install any of the other stack elements, and just use K8s as a container for Logstash (and metricbeat to monitor it), and to send to Elastic Cloud. In which case we'd need only the sections with `metadata.name: metricbeat` sections, and not any of the `filebeat`, `elasticsearch` or `kibana` sections. And we wouldn't need the `elasticsearch` and `kibana` modules in the `metricbeat` definition. 
And it might make sense to add this as a recipe to refer back to, to reduce the confusion with the rest of the file?

WDYT?

Reply:
I probably mess up the example when I replied. This should be valid to say CRDs as a prerequisite.

My understanding of ECK includes two parts. 1. CRD 2. operator. In order to have CRDs work with full functionality, we need both. The part I was uncertain is which features/configs need operator to work, so hesitate to just suggest installing CRD. But in this feature connecting to Elastic Cloud, I think we are safe to require CRD only.